config/locales/devise.pl.yml

```
@@ -0,0 +1,63 @@
+pl:
+  devise:
+    confirmations:
+      confirmed: "Okej, potwierdziliśmy Twój adres :)" #"Your email address has been successfully confirmed."
```

Please separate the `#` sign from the content of the comment with 1 space :wink:
```suggestion
      confirmed: "Okej, potwierdziliśmy Twój adres :)" # "Your email address has been successfully confirmed."
```